tests/Makefile

```diff
@@ -1,55 +1,55 @@
+PROG_NAME = testrom
```

Comment:
I'm going to submit the new build system this week (it's the first PR of the series of the audio library, given that the new build system contains RSP compiling rules). 

It doesn't matter whether this change is committed or not before that, but I'm skipping reviewing it as it would be overhauled anyway